snaps_k8s/provision/ansible_configuration.py

```diff
@@ -281,6 +275,37 @@ def launch_multus_cni(k8s_conf):
         })
 
 
+def create_cluster_role(k8s_conf):
+    """
+    This function is used to launch multus cni
+    """
+    master_host_name, master_ip = config_utils.get_first_master_host(k8s_conf)
+    logger.info('EXECUTING CREATE CLUSTER ROLE PLAY. Master ip - %s, '
+                'Master Host Name - %s', master_ip, master_host_name)
+    ret_val = ansible_utils.apply_playbook(consts.K8_MULTUS_SET_MASTER,
```

Comment:
No return value is ever returned. Exception will be raised if there is a problem and the check below will always fail. This is what is causing the unit test to fail.